packages/components/combobox/src/combobox.ts

```diff
@@ -915,13 +919,13 @@ export class Combobox<T = any, U = T> extends FormControlMixin(ScopedElementsMix
 
   #addSelectedGroup(): void {
     if (this.#useVirtualList) {
-      if (this.items[0].label === msg('Selected') && this.items[0].type === 'group') {
+      if (this.items[0].label === msg('Selected', { id: 'sl.common.selected' }) && this.items[0].type === 'group') {
```

Comment:
So `common` is fine for now, until it isn't :) So if it turns out people want to customize `common` translations differently, then we'd need to make the id more specific. But fine for now :)